site/assets/scss/_sidebar.scss

```diff
@@ -26,28 +27,28 @@
       text-decoration: if($link-hover-decoration == underline, none, null);
       background-color: rgba($bd-purple-bright, .1);
     }
+  }
 
-    // Indent if there's no submenu
-    &:only-child {
-      margin-left: 1.25rem;
-    }
+  button {
```

Comment:
I'd target the element with a class, personally.

Reply:
done